README.md

```diff
@@ -20,7 +20,9 @@ we do now have an experimental standalone binary release available, see ['Standa
 	- if you need `ssh` to work behind the proxy you also need [`proxytunnel`](http://proxytunnel.sourceforge.net/) installed (available as `proxytunnel` package for Ubuntu, for example)
   - `rsync`: >= 2.6.9 (required by `balena sync`)
 
-##### Windows Support
+#### Windows Support
+
+> Note that the fastest way to have the CLI running on **Windows** is by using the [Standalone Version](https://github.com/balena-io/balena-cli#standalone-install). You can follow [this video tutorial](https://www.youtube.com/watch?v=j3JoA1EINUA) to learn how to install it.
```

Comment:
Very nice, @dansku ! This is good and correct but also contradicts the following paragraphs that state that the standalone install is experimental and imply that the npm install is preferred. I have thus added a commit to your branch with a rewriting of those paragraphs. Please review too! :-)